Cargo.toml

```diff
@@ -125,70 +125,70 @@ zkaleido-sp1-adapter = { git = "https://github.com/alpenlabs/zkaleido", tag = "v
 
 # IMPORTANT: ensure alloy-* and revm packages are of the same version as inside reth dependency
 # reth dependencies:
-alloy-consensus = { version = "0.7.3", default-features = false }
-alloy-eips = { version = "0.7.3", default-features = false }
-alloy-genesis = { version = "0.7.3", default-features = false }
-alloy-network = { version = "0.7.3", default-features = false }
-alloy-primitives = { version = "0.8.11", default-features = false }
-alloy-rlp = { version = "0.3.10", default-features = false }
-alloy-rlp-derive = "0.3.10"
-alloy-rpc-types = { version = "0.7.3", features = [
+alloy-consensus = { version = "0.9.2", default-features = false }
+alloy-eips = { version = "0.9.2", default-features = false }
+alloy-genesis = { version = "0.9.2", default-features = false }
+alloy-network = { version = "0.9.2", default-features = false }
+alloy-primitives = { version = "0.8.19", default-features = false }
+alloy-rlp = { version = "0.3.11", default-features = false }
+alloy-rlp-derive = "0.3.11"
+alloy-rpc-types = { version = "0.9.2", features = [
   "eth",
 ], default-features = false }
-alloy-rpc-types-eth = { version = "0.7.3", default-features = false, features = [
+alloy-rpc-types-eth = { version = "0.9.2", default-features = false, features = [
   "serde",
 ] }
-alloy-serde = { version = "0.7.3", default-features = false }
-alloy-sol-types = "0.8.11"
+alloy-serde = { version = "0.9.2", default-features = false }
+alloy-sol-types = "0.8.19"
 # TODO: fix exact version during the next reth bump.
```

Comment:
Please remove this comment. 

P.S. Previously, at `1.1.3`, cargo automatically used `0.7.8` for `reth trie`, so it was required to use exact version.